src/domains/dashboard/components/Migration/MigrationBanner.tsx

```diff
@@ -41,7 +41,7 @@ export const MigrationBanner = () => {
 				</div>
 
 				<div className="hidden w-[304px] flex-shrink-0 pt-2 pb-4 md:block lg:w-[475px]">
-					<Image name="PolygonMigrationBanner" useAccentColor={false} />
+					<Image name="PolygonMigrationBanner" onlyLight />
```

Comment:
I think the `onlyLight` prop is a bit confusing, since it controls whether the accented images are shown when the dark theme is active or not. Could be implemented by passing in a dynamic value as the `useAccentColor` prop instead, e.g.

```ts
const { isDarkMode } = useTheme();

...

<Image useAccentColor={!isDarkMode} />
```

Reply:
@dated good one, updated